rpcs3/Loader/ELF.h

```diff
@@ -329,6 +324,21 @@ class elf_object
 		}
 	}
 
+	elf_object& operator=(elf_error error)
```

Comment:
Sad. Could just make set_error public. This operator indirectly violates the rule of 0/3/5. For instance, the elf object has its own implicit copy/move constructors and operators.
`xxx = elf_object{}` must be called explicitly if you want to clear it.

Reply:
Another tiny explicit method `clear` could do it.

Reply:
Done.